### DISCUSSION TITLE
Embedding and create discussion on the GitHub page

### DISCUSSION THREAD

**Discussion by BartDubois** [community/community#3334] [Discussions]
I would like to ask if it will be possible to embed the discussion on GitHub page. It would be nice addition to the feature of exposing pages through GitHub and link content with the topics.
It should also allow to create the topic automatically with relation to content page if it not exists yet.

**Comment by PranamBhat** [comment]
Hello @BartDubois 

**GitHub Pages** is a static site hosting service that takes HTML, CSS, and JavaScript files straight from a repository on **GitHub**, optionally runs the files through a build process, and publishes a website.

To be more specific, GitHub Pages will come into picture only when a particular user has to publish something from his own. There's no group/team concept. **GitHub Page** a subsidiary of **GitHub**. So an individual is always can post his query over **GitHub Discussion**.


Thank you!

**Reply by greggman** [reply]
This seems like a pretty tone deaf response.

It's common to have a live commenting system in documentation or tutorials for libraries. Often people use [disqus](https://disqus.com) for this. Now github provides a discussion system. It would be natural to want to be able to use that discussion system directly in the docs of some API being developed on github, embedded in the page, same as disqus.

If github wants to limit how people use it they could set it up so discussions can only be embedded in the github pages of the same repo but it's a totally reasonable request to want to be able to embed discussion in github pages.

**Reply by kaylumah** [reply]
Was searching for exactly this @greggman looking to replace Disqus as a commenting system over at my blog.
Sorry to hear this will not be considered

**Reply by greggman** [reply]
I sent in a feature request directly to github support.

> Many libraries and tutorials hotest on github pages allow embedded comments. Most of them use disqus.com for this. It seems it would be a natural fit to support using github discussions for this
>
> 1. being able to embed github discussions on github pages (including to ability to add a new comment)
> 2. being able to auto-generate new discussions for new github pages
>
> Please consider this a feature request.

the customer service rep replied

> Thank you very much for taking the time to write in with this request! I really like the idea and agree that Discussions could be great for that.
>
> Your ticket was automatically routed to our Developer Support team but our Product team is better suited to help you out with this kind of suggestion. I've forwarded your ticket over to that team, but please note that they cannot guarantee a response to every feature request submission.

So I suggest you request the feature too by contacting them directly. Lots of requests will hopefully allow this to happen.

**Reply by BartDubois** [reply]
@greggman do you mean by e-mail or some other way? I would like to push through the same channel as you.

**Reply by BartDubois** [reply]
> There's no group/team concept. **GitHub Page** a subsidiary of **GitHub**.

The GitHub Page are natural extension to the GitHub repository. Allowing to embed the discusion to the web pages is about bulding teams and comunity arout the project or concept. Allowing using GitHub ecosystem to create cosystent environment is the win-win situation.

The project benefits from the tool services, and the comunity is coupled to the tool. Anyway the participants will need to have GitHub accout to post the message.

I do not see where the fact that GitHub pages as the CSS/JS/HTML being the static web hosting can't be connect with GitHub ecosystem.

**Reply by greggman** [reply]
> @greggman do you mean by e-mail or some other way? I would like to push through the same channel as you.

I mean by clicking ["Contact Github"](https://support.github.com/contact) at the bottom of this page

**Comment by rufuspollock** [comment]
You may want to check out https://giscus.app

> A comments system powered by [GitHub Discussions](https://docs.github.com/en/discussions). Let visitors leave comments and reactions on your website via GitHub! Heavily inspired by [utterances](https://github.com/utterance/utterances).
